### DISCUSSION TITLE
Got error 200: OK.Something went wrong

### DISCUSSION THREAD

**Discussion by Bekcomp** [postaddictme/instagram-php-scraper#1074] [Q&A]
The scraper was working just fine, but suddenly I got this error:
Uncaught InstagramScraper\Exception\InstagramException: Response code is 200: OK.Something went wrong. Please report issue. 

I got it on every call after the login request.
I can't figure out how to fix this. 
I tried running the code from another server (different IP), run it on different devices and different locations.

Also it's strange that the code is 200, since this normally indicates that a request is ok.

**Comment by trialforce** [comment]
I just installed the code, make a basic account request and get the same result. Not working here.

**Reply by trialforce** [reply]
As i see we have error on "extractSharedDataFromBody" method.
